### PR TITLE
Implement topics needed by the UI for road weather use case

### PR DESCRIPTION
**Pull request by dan-du-car**
# PR Details
## Description
Implement topics needed by the UI by leveraging the existing UI layouts

## Motivation and Context
Road Weather use case

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: carma_wm_ctrl/src/WMBroadcaster.cpp
Comment (msmcconnell) [inline carma_wm_ctrl/src/WMBroadcaster.cpp:265]: The participant list you have here should be based on the TrafficControlParam.vclasses list. You can use the  participantsChecker() method to extract that from the control message. There is also a invertParticipants() method if that is needed.

File: carma_wm_ctrl/src/WMBroadcaster.cpp
Comment (msmcconnell) [inline carma_wm_ctrl/src/WMBroadcaster.cpp:902]: What is the difference in this outgoing message between .value and .advisory_speed?

**Review by msmcconnell — CHANGES_REQUESTED** [review]
This is looking really good Dan. Nice Job! Just 2 comments

File: carma_wm_ctrl/src/WMBroadcaster.cpp
Comment (dan-du-car) [reply carma_wm_ctrl/src/WMBroadcaster.cpp:902]: value is only used for Speed Limit event type. The Advisory speed is for the closed event type. The UI is taking data from different speed fields depends on the event types.

Comment (msmcconnell) [reply carma_wm_ctrl/src/WMBroadcaster.cpp:902]: ok sounds good

Comment (dan-du-car) [reply carma_wm_ctrl/src/WMBroadcaster.cpp:902]: https://github.com/usdot-fhwa-stol/carma-platform/pull/1197/commits/761f745a65683501b99f5387e17f2388de375e83

File: carma_wm_ctrl/src/WMBroadcaster.cpp
Comment (dan-du-car) [reply carma_wm_ctrl/src/WMBroadcaster.cpp]: https://github.com/usdot-fhwa-stol/carma-platform/pull/1197/commits/761f745a65683501b99f5387e17f2388de375e83

File: carma_wm_ctrl/src/WMBroadcaster.cpp
Comment (msmcconnell) [inline carma_wm_ctrl/src/WMBroadcaster.cpp:265]: Looking over the implementation, I don't think you need to invert the provided participants. The message specified which participants are restricted and your implementation of DigitalMinimumGap works the same way so I think a call to participantChecker() is sufficient.

**Review by msmcconnell — CHANGES_REQUESTED** [review]

File: carma_wm_ctrl/src/WMBroadcaster.cpp
Comment (dan-du-car) [reply carma_wm_ctrl/src/WMBroadcaster.cpp]: https://github.com/usdot-fhwa-stol/carma-platform/pull/1197/commits/b61c21edc31aa0b79c0e2e72ad8ddcd2f182faa8

**Review by msmcconnell — APPROVED** [review]
Looks good to me. Will wait for CI to show build is valid before merging

**Comment by msmcconnell** [comment]
This PR resolves 
https://github.com/usdot-fhwa-stol/carma-platform/issues/1205

**Pull request merged**
